### DISCUSSION TITLE
Working on Macbook Pro early 2015 - Big Sur

### DISCUSSION THREAD

**Discussion by kendalpercimoney** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#62] [General]
This seems to work on the MacBook Pro early 2015 model, although I had to close and reopen it to get it to work the first time. (when I reopened it, the bar was on 103%).

**Comment by thomas22122212** [comment]
I can confirm that. It's working as described on my MacBook Pro 13" Early 2015 on Big Sur 11.1

**Comment by cauafroes** [comment]
Apple has a built in setting in system preferences that does some battery preservative stuff too, does your Macs support it?

Not supported on 2010 machines like mine unfortunately.

**Reply by kendalpercimoney** [reply]
Yeah, this model doesn't have that option...
